x-pack/plugins/security_solution/server/lib/detection_engine/rule_response_actions/types.ts

```diff
@@ -19,6 +19,14 @@ export type Alert = ParsedTechnicalFields & {
   process?: {
     pid: string;
   };
+  kibana: {
```

Comment:
Why did you require to extend `Alert` type?

Reply:
Alerts are being extended to nested syntax (from dotted) and we need to specify some rule info here to pass further to our actions - for reference. Eg. in Response Actions History Log (outside of detections engine) we can link to a rule that triggered the alert and therefore the action.